internal/config/toml.go

```diff
@@ -0,0 +1,120 @@
+package config
+
+import (
+	"time"
+
+	"github.com/BurntSushi/toml"
+)
+
+// tomlConfig is the top-level TOML configuration fragment
+type tomlConfig struct {
+	Service *serviceSection
+	Etcd3   *etcd3Section
+	Lock    *lockSection
+}
+
+// serviceSection holds the optional `service` fragment
+type serviceSection struct {
+	Address *string
+	Port    *uint64
+}
+
+// etcd3Section holds the optional `etcd3` fragment
+type etcd3Section struct {
+	Endpoints    []string
```

Comment:
Do you see any value in adding toml struct tags to all values (even if the default behavior of lowercasing the field name is correct) for consistency when reading the code?